### PR TITLE
Created the getcount route that transforms the data into a json with …

### PR DESCRIPTION
**Pull request by xpandalord**
…various types of use-of-force and the relative count for each one

# Description
The getcount route was added. It returns the number of usages of each type of force.
# Number of Files changed
3
# Questions for Reviewer
None
# Type of Change
- [X] New Feature
- [ ] Feature Update
- [ ] Fix Error

# Status
- [X] Tested

### CODE REVIEW

**Review by nimu77 — COMMENTED** [review]
great work andy!

File: project/app/api/getcount.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+router = APIRouter()
+
+@router.get('/getcount')
+async def getcount():
+    '''
+    Get jsonified dataset from all_sources_geoed.csv
+    '''
+
+    # Path to dataset used in our endpoint
```
Comment (AuFeld) [inline project/app/api/getcount.py:19]: Good use of comments. Easy to follow in your code.

File: project/app/api/getcount.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import json
+import ast
+import app.api.getcounthelper as gch
+
+router = APIRouter()
+
+@router.get('/getcount')
+async def getcount():
+    '''
+    Get jsonified dataset from all_sources_geoed.csv
```
Comment (AuFeld) [inline project/app/api/getcount.py:16]: Good use of comments @xpandalord . Easy to follow in your code.

File: project/app/api/getcounthelper.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
+def helper(df):
+    count = {}
+    for set_of_tags in df["tags"]:
+        for tag in set_of_tags:
+            if tag in count:
+                count[tag] += 1
+            else:
+                count[tag] = 1
+
+    force = {}
```
Comment (AuFeld) [inline project/app/api/getcounthelper.py:10]: For the future, let's see some more commenting. This looks like 'our scale' re police brutatlity.

**Review by AuFeld — APPROVED** [review]
Looks good @xpandalord, however, let's see a consistent use of comments and documentation throughout your code.

**Review by nimu77 — APPROVED** [review]
Great work with defining types of force.

**Pull request merged**